tests/testSelectFluxMag0.py

```diff
@@ -25,30 +25,43 @@
 """Test lsst.obs.lsstSim.selectFluxMag0 and integration with pipe.tasks.scaleZeroPoint
 """
 import numpy
+import sys
 import unittest
 
-import lsst.daf.base
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
 import lsst.afw.math as afwMath
+import lsst.daf.base
 
-import lsst.utils.tests as utilsTests
 from lsst.daf.persistence import DbAuth
-from lsst.pipe.tasks.scaleZeroPoint import SpatialScaleZeroPointTask
 from lsst.obs.lsstSim.selectFluxMag0 import SelectLsstSimFluxMag0Task
+from lsst.pipe.tasks.scaleZeroPoint import SpatialScaleZeroPointTask
+import lsst.utils.tests
 
 
-#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
-
 class WrapDataId():
     """A container for dataId that looks like dataRef to computeImageScaler()
     """
     def __init__(self, dataId):
         self.dataId = dataId
 
+
 class ScaleLsstSimZeroPointTaskTestCase(unittest.TestCase):
     """A test case for ScaleLsstSimZeroPointTask
     """
+    def setUp(self):
+        """Initialize the DB connection.  Raise SkipTest if unable to access DB."""
+        config = SpatialScaleZeroPointTask.ConfigClass()
+        config.selectFluxMag0.retarget(SelectLsstSimFluxMag0Task)
+        print config
+        try:
+            DbAuth.username(config.selectFluxMag0.host, str(config.selectFluxMag0.port)),
+        except (Exception, RuntimeError) as e:
```

Comment:
`RuntimeError` is a subclass of `Exception` so you don't need to catch it. `Exception` is a bit general though.